tests/tt_eager/integration_tests/test_bert.cpp

```diff
@@ -33,6 +33,8 @@ Tensor encoder(Tensor&& hidden_states, const Tensor& attention_mask, const Param
         .in0_block_w = 4,
         .out_subblock_h = 4,
         .out_subblock_w = 2,
+        .out_block_h = 12,
+        .out_block_w = 8,
         .per_core_M = 12,
         .per_core_N = 8,
```

Comment:
We should probably clean this up:
- `in0_block_w` can be renamed `inner_dim_block` (or something that is not specific to in0)
- Pack all the `*_h` and `*_w` together?
- `per_core_M/N` should be renamed to `per_core_h/w` to be in line with the other parameters

Reply:
issue for tracking
https://github.com/tenstorrent/tt-metal/issues/15054